src/compiler/scala/tools/nsc/typechecker/Typers.scala

```
@@ -4480,6 +4480,13 @@ trait Typers extends Modes with Adaptations with Tags {
                   issue(accessibleError.get)
                   setError(tree)
                 }
+            case t if t.tpe.isErroneous && !qual.tpe.isErroneous =>
+              // Either error has been reported or we are dealing with 'invalid bounds' errors.
+              // The latter might be caused by forcing the info yet the real error reporting
+              // will be done in refchecks, so we delay it until then.
+              // qual.tpe check is done to avoid throwing spurious errors
+              // (see t692.scala for one example)
+              t setType sym.tpe.normalize
```

Re 1) since we have a silent for makeAccessible all the real errors are handled before. Although stabilize can be run afterwards, this is already outside of silent so it should report any errors.
Usually I am reluctant to put things like 'reporter.errors > 1'  as it seems hackish to me.

Re 2) good point.